src/Components/Web.JS/src/JSInitializers/JSInitializers.Server.ts

```diff
@@ -5,14 +5,21 @@ import { CircuitStartOptions } from '../Platform/Circuits/CircuitStartOptions';
 import { JSInitializer } from './JSInitializers';
 
 export async function fetchAndInvokeInitializers(options: Partial<CircuitStartOptions>) : Promise<JSInitializer> {
+  if (options.initializers) {
```

Comment:
Under what circumstances does this happen?

Reply:
Only when Blazor web is in charge of loading the initializers.